input/pagecontent/profiles.md

```diff
@@ -7,17 +7,26 @@ The following profiles have been defined for this implementation guide:
 * [CancerRelatedMedicationRequest]
```

Comment:
I recommend that the profiles, valuesets, etc pages are a waste of time to maintain and of very low utility to the reader.
Better to create groups in the sushi-config.yaml along the same lines (e.g., group the maturity level 2 and maturity level 0 profiles into separate groups in the sushi-config) and then have the menu link to sections of the artifacts.html page.   These pages, since they don't contain the descriptions of the resources, just their name, are MUCH LESS USEFUL TO THE READER than the content of the artifacts.html page.